GVRf/Framework/framework/src/main/java/org/gearvrf/GVRBillboard.java

```diff
@@ -0,0 +1,64 @@
+package org.gearvrf;
+import org.joml.Vector3f;
+
+public class GVRBillboard extends GVRBehavior
```

Comment:
Since this is a new public component it needs some javadoc. For example of what kind, see GVRSwitch. From the looks of it the class needs one, the constructor - very basic. It is unfortunate and unnecessary that we expose the onDrawFrame method so I guess you can put a javadoc saying "do not call this" or something.